setup.py

```diff
@@ -4,39 +4,22 @@
     long_description = fh.read()
 
 dependencies = [
-    "chia-blockchain==1.8.1",
-    "packaging==23.0",
```

Comment:
I don't see any removals in this PR that would make this unused.  Was it not used directly before anyways?

Reply:
It wasn't used, and afaik chia-blockchain will install it as a dep anyway

Reply:
If it wasn't used directly, that's fine.  We do want to describe all direct dependencies directly though, and not depend on getting them indirectly.

I did a quick search and agree that it appears to not be used directly here.  Feel free to close this conversation after reading this.